Assignment2/claubaldman-ClaudioBaldman.ttl

```diff
@@ -0,0 +1,22 @@
+@base <http://www.oeg-upm.net/ontologies/class/>
+@prefix class: <http://www.oeg-upm.net/ontologies/class#>
+@prefix sensor=<http://www.oeg-upm.net/ontologies/sensor#>
+@prefix computer=<http://www.oeg-upm.net/ontologies/computer#>
+@prefix measurement=<http://www.oeg-upm.net/ontologies/measurement#>
+@prefix user=<http://www.oeg-upm.net/ontologies/user#>
+   
+    :Class01
+                class:includes:Sensor029
+                class:includes:Computer101
+
+    :Sensor029
+                sensor:hasMeasurement:Measurement8401
+
+    :Computer101
+                computer:hasOwner:User10A
+
+    :Measurement8401
+                measurement:hasTemperature:"29"^^xsd:integer

```

Comment:
Still not valid. It should be measurement:hasTemperature "29"^^xsd:integer
Similar in the other lines below.